### ISSUE TITLE
How to use multiple modifiers on the same JSX Eventhandler?

### ISSUE THREAD

**Issue by chanlito** [wonderful-panda/vue-tsx-support#10]
The following isn't working

```ts
...
onKeyup={m.enter(this.addTodo)}
onKeyup={m.esc(this.addTodo)} // JSX can't have 2 onKeyup
...
```

Can we some how chain the modifiers? like `onKeyup={m.enter.esc(this.addTodo)}` ?

**Comment by wonderful-panda** [comment]
[keys](/wonderful-panda/vue-tsx-support#keys) is available from v2.2.0.

```typescript
<div onKeyup={m.keys("enter", "esc")(this.addTodo)} />
```

**Comment by chanlito** [comment]
Awesome. 🎉
